### ISSUE TITLE
Don't assume that "$" == jQuery

### ISSUE THREAD

**Issue by vegantriathlete** [glad/glDatePicker#66]
In your initial function declaration pass the argument of "$" and declare it as jQuery at the end.

For example, I am suggesting that you wrap your code as follows:

<code>;(function($) { . . .  })(jQuery);</code>

**Comment by DiscoNova** [comment]
That is done in order to _ascertain_ that within the function (ie. within this library) whenever reference is made to $, what it actually refers to is jQuery, no matter what "jQuery" is called outside of the library (or, for that matter, no matter what $ refers to outside the library).

Edit: just to make it clear; I am agreeing with vegantriathlete. On a second read of my message it looked liked exactly the other way round :)

**Comment by vegantriathlete** [comment]
Yes, thanks for the edit. I thought that you were disagreeing with me (when I read the pre-edit version of the post) and I was quite confused.

**Comment by FelixSchwarz** [comment]
@vegantriathlete Probably your initial comment should be rephrased to make it clear you're proposing a solution (not showing existing code from the library). Also maybe a new issue summary would be good, e.g. "glDatePicker is not compatible with jQuery.noConflict() - don't assume that "$" == jQuery".

**Comment by vegantriathlete** [comment]
@FelixSchwarz I've updated the initial comment. I hope it clarifies things. I don't know where / how to create an issue summary, but I think that what you have suggested it perfectly fine.

**Comment by blittle** [comment]
This is a big problem with a very minor fix. Please considering adding it :)
